ui/src/DashboardTable.js

```diff
@@ -0,0 +1,416 @@
+        }),
+      }}
+    >
+      {numSelected > 0 ? (
+        <Typography
+          sx={{ flex: '1 1 100%' }}
+          color="inherit"
+          variant="subtitle1"
+          component="div"
+        >
+          {numSelected} selected
+        </Typography>
+      ) : (
+        <Typography
+          sx={{ flex: '1 1 100%' }}
+          variant="h6"
+          id="tableTitle"
+          component="div"
+        >
+
+        </Typography>
+      )}
+
+      {numSelected > 0 && (
+        <Tooltip title="Delete">
+          <IconButton>
+            <DeleteIcon onClick={onDelete} />
+          </IconButton>
+        </Tooltip>
+      )}
+    </Toolbar>
+  );
+}
+
+EnhancedTableToolbar.propTypes = {
+  numSelected: PropTypes.number.isRequired,
+};
+
+export default function DashboardTable({ models }) {
+  const [rows, setRows] = React.useState([])
+  const [order, setOrder] = React.useState('asc');
+  const [orderBy, setOrderBy] = React.useState('');
+  const [selected, setSelected] = React.useState([]);
+  const [page, setPage] = React.useState(0);
+  const [dense, setDense] = React.useState(false);
+  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+  const [count, setCount] = React.useState(0);
+  const [deleted, setDeleted] = useState('');
+  const [progress, setProgress] = useState(0);
+
+  const splittedUrl = window.location.href.split("/");
+
+  const [sandboxCommit, setSandbox] = useState(
+    splittedUrl.includes("sandbox") ?
+      splittedUrl.at(-1) : false);
+
+  useEffect(() => {
+    fetch(`${FETCH_PATH}/api/models_paginated/${page * rowsPerPage}/${page * rowsPerPage + rowsPerPage}`)
+      .then((response) => response.json())
+      .then((json) => {
+        setRows(json["models"]);
+        setCount(json["count"]);
+        json["models"].map((m) => {
+          if (m.progress < 100) {
+            setProgress(progress + m.progress + 3);
+          }
+        })
+      });
+  }, [page, rowsPerPage, progress]);
+
+
+  const handleSelectAllClick = (event) => {
+    if (event.target.checked) {
+      const newSelected = rows.map((n) => n.id);
+      setSelected(newSelected);
+      return;
+    }
+    setSelected([]);
+  };
+
+  const handleClick = (event, filename) => {
+    const selectedIndex = selected.indexOf(filename);
+    let newSelected = [];
+
+    if (selectedIndex === -1) {
+      newSelected = newSelected.concat(selected, filename);
+    } else if (selectedIndex === 0) {
+      newSelected = newSelected.concat(selected.slice(1));
+    } else if (selectedIndex === selected.length - 1) {
+      newSelected = newSelected.concat(selected.slice(0, -1));
+    } else if (selectedIndex > 0) {
+      newSelected = newSelected.concat(
+        selected.slice(0, selectedIndex),
+        selected.slice(selectedIndex + 1),
+      );
+    }
+
+    setSelected(newSelected);
+  };
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeDense = (event) => {
```

Comment:
Not used

Reply:
removed in https://github.com/AECgeeks/ifc-pipeline-validation/pull/51/commits/c181fbc1ee91f68009e457f01232712646753679